en/getting-started/install/index.md

```diff
@@ -0,0 +1,70 @@
+>
+>       The default installation path of the application is C:\CocosDashboard, you can change it during the installation.
+>
+> 2. If there is an error about the digital signature of CAB file, please try to install with administrator privileges.
+> 
+> 3. For some Windows PCs that are very old and have been on the intranet for a long time or have not been updated with OS patches for a long time, there may be some errors caused by missing dlls when running, try to install the system patch to solve: <https://support.microsoft.com/en-us/help/2999226/update-for-universal-c-runtime-in-windows>.
+> 
+> 4. If the "Access Denied" pop-up window appears during installation, please make sure that the operating system installed on your machine is the official version of Microsoft, not a customized or streamlined third-party version.
+
+### Installation Instructions for MacOS Version
+
+The installer for Cocos Dashboard for MacOS is a dmg image file. Double-click the dmg file, then drag and drop **CocosDashboard.app** into your **Applications** folder shortcut, or any other location. Then double-click the dragged **CocosDashboard.app** and you're ready to go.
+
+> **Note**.
+> 1. If you can't open it after downloading, it indicates that the dmg or app file is corrupted, from an unknown developer, or contains malware, etc. Please right-click the dmg or app file in the Finder and select **Open**, then click **Open** again in the pop-up dialog box. Then please go to **System Preferences -> Security & Privacy** and click **Open Anyway** so that it can be launched normally later.
+>
+> 2. If you get a "corrupted, can't open" message during the installation process, you need to check if there is any software like Xcode occupying the files in the Dashboard installation directory. If so, exit, then uninstall Dashboard and reinstall it.
+
+### Operating System Requirements
+
+The supported system environments for Cocos Dashboard are
+
+The minimum supported version of Mac OS X is OS X 10.9.
+The minimum supported version of Windows is Windows 7 64-bit.
+
+## Running Cocos Dashboard
+
+On Windows, double-click the **CocosDashboard.exe** file in the `CocosDashboard` folder after unzipping to start Cocos Dashboard.
+
+On MacOS, double-click the dragged out **CocosDashboard.app** application icon to launch Cocos Dashboard.
+
+You can set a Quick Launch, Dock, or shortcut to the entry file as you like, so you can run it whenever you want.
+
+### Checking the Graphics Card Driver
+
+For some Windows operating systems and graphics card models, you may encounter error message like
+
+```
+This browser does not support WebGL...
+```
+
+This is due to the editor relying on GPU rendering, which is not supported by the graphics card driver. If this happens, you can usually fix it by making sure that the official driver for your graphics card is successfully installed.
```

Comment:
```suggestion
This is due to the editor relying on GPU rendering, which is not supported by the graphics card driver. If this happens, it can usually be fixed by making sure that the official driver for your graphics card is successfully installed.
```